frame/example/src/lib.rs

```diff
@@ -84,12 +84,13 @@
 //!
 //! \## Terminology
 //!
-//! // Add terminology used in the custom pallet. Include concepts, storage items, or actions that you think
-//! // deserve to be noted to give context to the rest of the documentation or pallet usage. The author needs to
-//! // use some judgment about what is included. We don't want a list of every storage item nor types - the user
-//! // can go to the code for that. For example, "transfer fee" is obvious and should not be included, but
-//! // "free balance" and "reserved balance" should be noted to give context to the pallet.
-//! // Please do not link to outside resources. The reference docs should be the ultimate source of truth.
+//! // Add terminology used in the custom pallet. Include concepts, storage items, or actions that
+//! you think // deserve to be noted to give context to the rest of the documentation or pallet
+//! usage. The author needs to // use some judgment about what is included. We don't want a list of
+//! every storage item nor types - the user // can go to the code for that. For example, "transfer
+//! fee" is obvious and should not be included, but // "free balance" and "reserved balance" should
+//! be noted to give context to the pallet. // Please do not link to outside resources. The
+//! reference docs should be the ultimate source of truth.
```

Comment:
The diffs of comments in this file should probably be reverted or fixed; they're rewraps which don't preserve comment expectations such as "the `//` goes at the start of the line".

Reply:
Yeah I seem to have messed up with line wrapping.